src/sql-parser/src/ast/defs/expr.rs

```diff
@@ -180,7 +180,7 @@ pub enum Expr<T: AstInfo> {
     /// `<expr>[<expr>]`
```

Comment:
If I'm understanding this correctly, you may want to update this comment to something like `<expr>[<expr>]([<expr>])*`

Reply:
Good catch, ty